test/calibration/test_calibrations.py

```diff
@@ -1520,6 +1588,21 @@ def test_save_load_library(self):
             BackendData(backend).drive_freqs[0],
         )
 
+    def test_json_round_trip(self):
+        """Test round trip test for JSON file format.
+
+        This method guarantees full equality including parameterized template schedules
+        and we can still generate schedules with loaded calibration instane,
+        even though calibrations is instantiated outside built-in library.
+        """
+        self.cals.save(file_type="json", overwrite="True", file_prefix=self._prefix)
+        loaded = self.cals.load(file_path=self._prefix + ".json")
+        self.assertEqual(self.cals, loaded)
```

Comment:
These three lines make a small, easy, and repeatable test. In `test_calibrations.py` we have several classes the create a `self.cals` in the setup. Can you add such a test to each of the classes that create `self.cals` in the setup? This might help catch strange errors.

Reply:
Added several tests in [895017a](https://github.com/Qiskit/qiskit-experiments/pull/1120/commits/895017abda231212e834763628b849f7d41488d7). As far as I can see most of test classes are tied to single calibration instance, i.e. `self.cals`. Instead of adding method something like `self.test_roundtrip_calibration` and provide new calibration test base class, I just duplicated the test code to each class.